src/renderers/shaders/ShaderChunk.js

```diff
@@ -1 +1,217 @@
+import roughnessmap_pars_fragment from './ShaderChunk/roughnessmap_pars_fragment.glsl';
+import shadowmap_pars_fragment from './ShaderChunk/shadowmap_pars_fragment.glsl';
+import shadowmap_pars_vertex from './ShaderChunk/shadowmap_pars_vertex.glsl';
+import shadowmap_vertex from './ShaderChunk/shadowmap_vertex.glsl';
+import shadowmask_pars_fragment from './ShaderChunk/shadowmask_pars_fragment.glsl';
+import skinbase_vertex from './ShaderChunk/skinbase_vertex.glsl';
+import skinning_pars_vertex from './ShaderChunk/skinning_pars_vertex.glsl';
+import skinning_vertex from './ShaderChunk/skinning_vertex.glsl';
+import skinnormal_vertex from './ShaderChunk/skinnormal_vertex.glsl';
+import specularmap_fragment from './ShaderChunk/specularmap_fragment.glsl';
+import specularmap_pars_fragment from './ShaderChunk/specularmap_pars_fragment.glsl';
+import tonemapping_fragment from './ShaderChunk/tonemapping_fragment.glsl';
+import tonemapping_pars_fragment from './ShaderChunk/tonemapping_pars_fragment.glsl';
+import uv_pars_fragment from './ShaderChunk/uv_pars_fragment.glsl';
+import uv_pars_vertex from './ShaderChunk/uv_pars_vertex.glsl';
+import uv_vertex from './ShaderChunk/uv_vertex.glsl';
+import uv2_pars_fragment from './ShaderChunk/uv2_pars_fragment.glsl';
+import uv2_pars_vertex from './ShaderChunk/uv2_pars_vertex.glsl';
+import uv2_vertex from './ShaderChunk/uv2_vertex.glsl';
+import worldpos_vertex from './ShaderChunk/worldpos_vertex.glsl';
+
+import cube_frag from './ShaderLib/cube_frag.glsl';
+import cube_vert from './ShaderLib/cube_vert.glsl';
+import depth_frag from './ShaderLib/depth_frag.glsl';
+import depth_vert from './ShaderLib/depth_vert.glsl';
+import distanceRGBA_frag from './ShaderLib/distanceRGBA_frag.glsl';
+import distanceRGBA_vert from './ShaderLib/distanceRGBA_vert.glsl';
+import equirect_frag from './ShaderLib/equirect_frag.glsl';
+import equirect_vert from './ShaderLib/equirect_vert.glsl';
+import linedashed_frag from './ShaderLib/linedashed_frag.glsl';
+import linedashed_vert from './ShaderLib/linedashed_vert.glsl';
+import meshbasic_frag from './ShaderLib/meshbasic_frag.glsl';
+import meshbasic_vert from './ShaderLib/meshbasic_vert.glsl';
+import meshlambert_frag from './ShaderLib/meshlambert_frag.glsl';
+import meshlambert_vert from './ShaderLib/meshlambert_vert.glsl';
+import meshphong_frag from './ShaderLib/meshphong_frag.glsl';
+import meshphong_vert from './ShaderLib/meshphong_vert.glsl';
+import meshphysical_frag from './ShaderLib/meshphysical_frag.glsl';
+import meshphysical_vert from './ShaderLib/meshphysical_vert.glsl';
+import normal_frag from './ShaderLib/normal_frag.glsl';
+import normal_vert from './ShaderLib/normal_vert.glsl';
+import points_frag from './ShaderLib/points_frag.glsl';
+import points_vert from './ShaderLib/points_vert.glsl';
+import shadow_frag from './ShaderLib/shadow_frag.glsl';
+import shadow_vert from './ShaderLib/shadow_vert.glsl';
+
+export var ShaderChunk = {
```

Comment:
This conserves ShaderChunk as mutable. Are there existing uses or valid use cases for adding to ShaderChunk? Otherwise, AFAICT this export could be `export *;`? @Rich-Harris : Would that affect tree-shaking of individual shaders?